### DISCUSSION TITLE
question: Is there an Obtainium like app for Linux?

### DISCUSSION THREAD

**Discussion by abmusse** [ImranR98/Obtainium#777] [Q&A]
Awesome app!

Is there an Obtanium like app for Linux?

There are several projects that I would like to install via Github Releases that are not included in my standard linux repos.
I could just use the Github API and download the latest release via shell script but would much rather have an app like Obtainium manage those packages.

**Comment by roadkell** [comment]
This works quite well for me: https://github.com/marcosnils/bin/

**Reply by abmusse** [reply]
Interesting tool! Thanks for the suggestion!

**Reply by roke-julian-lockhart** [reply]
> https://github.com/ImranR98/Obtainium/discussions/777#discussioncomment-11019465

@Welved, might you elaborate how it doesn't work? As an example, can you not merely replace a URI to a package posted to GitHub releases with one posted to GitLab?

**Reply by roke-julian-lockhart** [reply]
> https://github.com/ImranR98/Obtainium/discussions/777#discussioncomment-11019602

@Welved, https://github.com/marcosnils/bin/blob/94bbdcac69f74abb8b3d9e5dcc0fcb22d72d6782/README.md#usage appears to indicate that there's some magic in `bin` that allows it to choose which *file* to download when you provide it with a *release*'s URI, whereas you're directly providing an archive-file URI (albeit like one would reasonably expect to).

Consequently, I suppose you must be correct - what a shame. Though, considering that Obtanium solely supports APKs (not even AABs) which are standardised, instead of collections of files encapsulated in myriad archive file types, I'm not surprised.

**Comment by ImranR98** [comment]
This is the same as #1053. Closing it in favour of that one (which is newer but has more suggestions).
